README.md

```diff
@@ -75,6 +76,22 @@ Use `o-teaser-collection--numbered` to number the list of teasers in the collect
 
 Use `o-teaser-collection--special` to add a darker background across the full width of the containing relative element, see an [example in the registry](http://registry.origami.ft.com/components/o-teaser-collection#demo-special).
 
+
+## Migration guide
+
```

Comment:
We should have a subheader which tells the reader which versions this migration guide accounts for like we do in other component readmes such as [o-footer](https://github.com/Financial-Times/o-footer#upgrading-from-3xx-to-4xx)